ci/04-run-test-suite.sh

```diff
@@ -3,17 +3,16 @@ set -e
 
 if [ -f ${HOME}/.bashrc ]; then
     source ${HOME}/.bashrc
-else
+elif [ -f ${HOME}/.bash_profile ]; then
     source ${HOME}/.bash_profile
 fi
 export VAEX_SERVER_OVERRIDE='{"dataframe.vaex.io":"dataframe-dev.vaex.io"}'
-pytest tests\
-        packages/vaex-core/vaex/datatype_test.py\
-        packages/vaex-core/vaex/file/\
-        packages/vaex-core/vaex/test/dataset.py::TestDataset\
-        --doctest-modules\
-            packages/vaex-core/vaex/datatype.py\
-            packages/vaex-core/vaex/utils.py\
-            packages/vaex-core/vaex/struct.py\
-            packages/vaex-core/vaex/groupby.py\
-        --timeout=1000
+python -m pytest --pyargs --doctest-modules --timeout=1000\
```

Comment:
I added `python -m` for exactly that reason. maybe the `source` commands above are overwriting `PATH` with something new? I can imagine removing those source commands might fix 'no module named pytest'